linera-service/src/cli_wrappers/docker.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::util::CommandExt;
+use anyhow::{Context, Result};
+use pathdiff::diff_paths;
+use std::path::PathBuf;
+use tokio::process::Command;
+
+pub struct DockerImage {
+    pub name: String,
```

Comment:
Does it need to be `pub`? Could alternatively provide a getter `fn name(&self) -> &str { .. }`. (See next comment)